docs/_themes/clusterhq/layout.html

```diff
@@ -0,0 +1,192 @@
+                        <img src="{{ pathto('_static/menu@2x.png', 1) }}" width="60" height="47" class="pull-right"/>
+                    </a>
+                </div>
+                <nav class="clearfix">
+                    <div class="col-sm-12 col-md-9">
+                        <ul class="menu hidden-xs">
+                            <li><a href="https://docs.clusterhq.com/en/latest/gettingstarted/index.html">Getting Started</a></li>
+                            <li class="current-menu-item"><a href="https://docs.clusterhq.com/en/latest/">Docs</a></li>
+                            <li><a href="https://clusterhq.com/solutions/">Solutions</a></li>
+                            <li><a href="https://clusterhq.com/about/">About</a></li>
+                            <li><a href="https://clusterhq.com/careers/">Careers</a></li>
+                            <li class="button menu hidden-xs"><a href="https://docs.clusterhq.com/en/latest/gettingstarted/index.html">Try Flocker</a></li>
+                        </ul>
+                    </div>
+                </nav>
+            </div>
+        </div>
+    </div>
+    
+    <!-- Contents -->
+    <div class="container contents">
+        <div class="row">
+            <!-- Contents menu -->
+            <div class="col-sm-3 page contents-menu">
+                <div class="visible-xs-block mob-contents">
+                    Contents
+                    <div class="pull-right plus">
+                        <a href="javascript:$('.contents-menu>ul').slideToggle();">+</a>
+                    </div>
+                </div>
+                <!-- Menu area start -->
+                {{ toctree(collapse=false, maxdepth=4) }}
+                <!-- Menu area end -->
+                <script>
+                $( window ).resize(function() {
+                    if($( window ).width() >= 768) {
+                        $('.contents-menu>ul').show();
+                    }
+                });
+                </script>
+            </div>
+            
+            <!-- Main contents area -->
+            <div class="col-sm-8 col-sm-offset-1 page white-bg">
+                {%- if pagename != "search" %}
+                <form class="search" action="{{ pathto('search') }}" method="get" _lpchecked="1">
+                    <input type="text" name="q" placeholder="Type here to search docs..." id="search">
+                    <input type="hidden" name="check_keywords" value="yes">
+                    <input type="hidden" name="area" value="default">
+                </form>
+                {%- endif %}
+                <!-- DOCS PAGE DATA HERE -->
+                {%- block document %}
+                        <script>
+                            var release_re = /^[0-9]+\.[0-9]+\.[0-9]+$/
+                            if (!'{{ release }}'.match(release_re)) {
+                                document.write(
+                        '<div class="admonition note warning"> \
```

Comment:
The issue I mentioned still remains. If you are in a rush to merge this, please create a follow-up issue. I guess it'll be a font change.